compiler/parser/chapel.lex

```diff
@@ -762,7 +762,8 @@ static int processBlockComment(yyscan_t scanner) {
         else if(fDocsCommentLabel[0] == '/' && fDocsCommentLabel[1] == '*') {
           depth--;
           d = 1;
-          USR_WARN("chpldoc comment not closed, ignoring comment: %s in Line no: %d\nThis comment is not closed properly", fDocsCommentLabel, startLine);
+          warning = true;
+          //USR_WARN("Bad comment: %s\nLine no: %d\nThis comment is not closed properly", stringBuffer.c_str(), startLine);
```

Comment:
Hmm, maybe it would be better to match the error style of the rest of the file, rather than the general USR_WARN that we use in the rest of the file.  See line 782 in this file